fbchat/get.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: UTF-8 -*-
+
+from __future__ import unicode_literals
+from .base import Base
+
+
+class Get(Base):
+    """Enables retrieving information about threads and messages"""
```

Comment:
A `Fetch` class would also be made, where most of these methods would have a corresponding `fetch_X` method.